extensions/amp-sidebar/0.1/amp-sidebar.js

```diff
@@ -198,9 +198,11 @@ export class AmpSidebar extends AMP.BaseElement {
   /** @override */
   onLayoutMeasure() {
     if (this.isToolbarExperimentEnabled_) {
-      // Check our toolbars for changes
-      this.toolbars_.forEach(toolbar => {
-        toolbar.onLayoutChange(() => this.onToolbarOpen_());
+      this.getAmpDoc().whenReady().then(() => {
+        // Check our toolbars for changes
+        this.toolbars_.forEach(toolbar => {
```

Comment:
@camelburrito Can this move to `buildCallback`?

Reply:
No because at this point the toolbar had been built and this code attempts to inject it at the right place on the DOM when layout changes based on the viewport width + toolbar config